install.sh

```diff
@@ -1,7 +1,19 @@
-cargo build --release
-sudo cp target/release/mobydick $PREFIX/bin/xyz.gelez.mobydick
-sudo cp *.appdata.xml $PREFIX/share/appdata/
-sudo cp *.desktop $PREFIX/share/applications/
+#!/usr/bin/env sh
+if [ -z "$PREFIX" ]; then
+  PREFIX=/usr/local
+fi
+
+if [ -f target/release/mobydick ]; then
```

Comment:
Isn't it missing a `!`? It seems to me you are building only if there is already the file.

Reply:
Yes, it was missing.